apis/test_config.py

```diff
@@ -155,6 +179,34 @@ def test_script(self) -> str:
     return '\n'.join(self.run_model_cmds)
 
 
+@attrs.define
+class GpuVmTest(TestConfig[Gpu]):
+  """Test config that runs on a single Cloud GPU VM instance.
+
+  Attributes:
+    test_name: Unique name for this test/model.
+    set_up_cmds: List of commands to run once when GPU is created.
+    run_model_cmds: List of commands to run the model under test.
+  """
+
+  test_name: str
+  vm_duration: str
```

Comment:
What's the difference between `vm_duration` and `time_out_in_min`? Can we directly use that?

Also miss the comment in the attributes if they are difference.

Reply:
`vm_duration` has been removed in later changes.